File: LC/lc485/lc485.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        res = 0
        out = 0
        for x in nums:
            if x:
                res += 1
            elif res:
                out = max(out, res)
                res = 0
        return max(out,res)
        

File: LC/lc485/lc485.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
  let mem = 0, cont = 0;
  for (i = 0; i < nums.length; i++) {
      if (nums[i]) {
          cont += 1;
          mem = (mem > cont) ? mem : cont;
      } else {cont = 0;}
  }
  return mem;
};